puppet/lib/puppet/face/node/status.rb

```diff
@@ -19,7 +19,7 @@
 
       args.map do |node|
         begin
-          response = Puppet::Util::Puppetdb::Http.action("/v4/nodes/#{CGI.escape(node)}") do |http_instance, path|
+          response = Puppet::Util::Puppetdb::Http.action("/pdb/query/v4/nodes/#{CGI.escape(node)}") do |http_instance, path|
```

Comment:
Before we could just set a top-level prefix but now we have two prefixes the terminus needs to know about